### ISSUE TITLE
clickhouse keeper failing

### ISSUE THREAD

**Issue by olexiyb** [duyet/charts#158]
I follow https://artifacthub.io/packages/helm/duyet/clickhouse-keeper

```
Error: YAML parse error on clickhouse-keeper/templates/deployment.yaml: error converting YAML to JSON: yaml: line 62: could not find expected ':'
```

**Comment by duyet** [comment]
Hi @olexiyb, could you give me your values.yaml file?

**Comment by olexiyb** [comment]
I have set only, I generated
```
helm --version 0.1.1 show values duyet/clickhouse-keeper > clickhouse-keeper.original.yaml
```
Copied file
```
cp clickhouse-keeper.original.yaml clickhouse-keeper.yaml
```

Next I have modified only

```
values:
   persistence:
     enabled: true
     storageClass: "local-path"
```

And after run
```
helm --version 0.1.1  -n clickhouse-keeper upgrade --install clickhouse-keeper duyet/clickhouse-keeper --create-namespace -f clickhouse-keeper.yaml
```

I use k3s 1.26

**Comment by duyet** [comment]
Hi @olexiyb, please review my fix with version 0.1.2.

```bash
helm --version 0.1.2  -n clickhouse-keeper upgrade --install clickhouse-keeper duyet/clickhouse-keeper --create-namespace -f clickhouse-keeper.yaml
```
